chia/util/condition_tools.py

```diff
@@ -159,6 +159,7 @@ def conditions_for_solution(
 ) -> Tuple[Optional[Err], Optional[List[ConditionWithArgs]], uint64]:
     # get the standard script for a puzzle hash and feed in the solution
     try:
+        # TODO: Note this call to Program.run_with_cost
```

Comment:
Note that we call `Program.run_with_cost`, which is `chia.types.program.run_program` in chia/util/condition_tools.py